src/localize/languages/it.json

```diff
@@ -2,83 +2,83 @@
-    "required": null,
-    "recommended": null,
-    "override_headline": null,
-    "hide_when_no_warning": null,
-    "hide_caption": null,
-    "disable_swiper": null,
+    "entity": "Entità",
+    "integration": "Integrazione",
+    "required": "Richiesto",
+    "recommended": "Consigliato",
+    "override_headline": "Sovrascrivere l'intestazione fornita dall'integrazione",
+    "hide_when_no_warning": "Nascondi quando non ci sono avvisi",
+    "hide_caption": "Nascondi la descrizione",
+    "disable_swiper": "Disabilità lo swipe delle allerte",
     "description": {
-      "start": null,
-      "current_expected": null,
-      "slots": null,
-      "separate_events": null,
-      "warning_watch_statement": null,
-      "end": null
+      "start": "L'integrazione selezionata crea più entità.",
+      "current_expected": "La prima con gli eventi attuali e la seconda con quelli futuri.",
+      "slots": "L'esatta quantità di è determinata dalla sua configurazione.",
+      "separate_events": "Ogni evento ha un'entità propria e separata.",
+      "warning_watch_statement": "Esistono 3 entità: avvertenze, considerazioni e avvisi/messaggi.",
+      "end": "Dovreste aggiungerle tutte per essere sicuro di non perdere nessun avviso."
     },
     "error": {
-      "expected_entity": null,
-      "too_many_entities": null,
-      "duplicate": null
+      "expected_entity": "L'entità di avviso futuro non è stata inserita. Gli eventi futuri non saranno mostrati.",
+      "too_many_entities": "Più entità del previsto non sono state fornite. Entita Previste: {expected}; Entita Ottenute {got} entità.",
+      "duplicate": "Trovati duplicati nell'elenco delle entità."
     }
   },
   "events": {
     "no_warnings": "Nessun allarme",
-    "nuclear": null,
-    "tornado": null,
-    "hurricane": null,
-    "wind": "vento",
-    "earthquake": null,
-    "volcano": null,
-    "snow_ice": "neve/ghiaccio",
-    "fog": "nebbia",
-    "high_temperature": null,
-    "low_temperature": "basse temperature",
-    "coastal_event": "eventi costieri",
-    "tsunami": null,
-    "sea_event": null,
-    "forest_fire": "incendi boschivi",
-    "avalanches": "valanghe",
-    "thunderstorms": "temporali",
-    "rain": "pioggia",
-    "flooding": null,
-    "air_quality": null,
-    "dust": null
+    "nuclear": "Nucleare",
+    "tornado": "Tornadi",
+    "hurricane": "Uragani",
+    "wind": "Vento",
+    "earthquake": "Terremoti",
+    "volcano": "Eruzione Vulcanica",
+    "snow_ice": "Neve/Ghiaccio",
+    "fog": "Nebbia",
+    "high_temperature": "Alte Temperature",
+    "low_temperature": "Basse temperature",
+    "coastal_event": "Eventi Costieri",
+    "tsunami": "Tsunami",
+    "sea_event": "Eventi Marini",
+    "forest_fire": "Incendi Boschivi",
+    "avalanches": "Valanghe",
+    "thunderstorms": "Temporali",
+    "rain": "Pioggia",
+    "flooding": "Allagamenti",
+    "air_quality": "Qualità dell'Aria",
+    "dust": "Tempesta di Sabbia/Polveri"
```

Comment:
Maybe only "**Polveri**" ? (that should be related to pollution, not to sand storms)

Reply:
For this one maybe better elucidation from @MrBartusek

Reply:
Yup dust is actually for stuff like blowing dust, sandstorms and such. It's Canada and USA thing

![image](https://user-images.githubusercontent.com/23432278/186041859-88294a70-ce7e-4318-80f2-dd72e5020613.png)

Reply:
Then it is already correct.

Reply:
Then it is already correct.